### PR TITLE
Merge ginkgo.master

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ log/
 .redcar/
 /nbproject
 .idea/
+
+.ruby-version
```
Comment (caesar2164) [inline .gitignore:35]: is there meant to not be a trailing newline here?

**Review by caesar2164 — APPROVED** [review]
I may not be the ideal person to review this, but it looks pretty reasonable...

File: .gitignore (hunk 1 above)
Comment (stvstnfrd) [reply .gitignore:35]: We didn't write any of this.

Comment (caesar2164) [reply .gitignore:35]: oh, duh! then ignore my comment!

**Pull request merged**